data/specs.json

```diff
@@ -23,6 +25,7 @@
     "int4_tensor_core_sparsity": { "full_name": "INT4 Tensor Core with Sparsity", "unit": "TOPS" },
     "manufacturer": { "full_name": "Manufacturer" },
     "architecture": { "full_name": "Architecture" },
+    "technology": { "full_name": "Technology" },
```

Comment:
To comply with NVIDIA data sheets, I suggest to name this field "Manufacturing Process" (process).